src/ShowPage/ShowPage.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { BrowserRouter, Route, Link } from 'react-router-dom';
+import { getSingleMovie } from '../apiCalls'
+import './ShowPage.css';
+
+
+
+
+// let allGenres;
+
+class ShowPage extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      currentMovie: {},
+      error: ''
+    }
+  }
+
+  componentDidMount() {
+    this.determineSingleMovie(this.props.id)
+  }
+
+
+  determineSingleMovie = (id) => {
+    getSingleMovie(id)
+    .then(singleMovie => this.setState({currentMovie: singleMovie}))
+    .catch(error => this.setState({ error: error }));
+  }
+
+    getGenres = () => {
+      return this.props.genres.map(genre => {
+        return <p>genre.name</p>
+      })
+    }
+
+    render() {
```

Comment:
As of now, we are not getting these `console logs` in the browser's console, nor are we getting any of the JSX from the `return` in `render.`

Reply:
Remember to use React Dev Tools to check on state and props! Console logs often lag behind the actual component data!

Reply:
We've solved this issue, by the way!